scripts/lib/install

```diff
@@ -123,22 +132,8 @@ EOF
         exit 1
 esac
 
-if [ "$os_id" = Ubuntu ] && ! apt-cache policy |
-           grep -q "^     release v=$os_release,o=Ubuntu,a=$os_codename,n=$os_codename,l=Ubuntu,c=universe"; then
-    set +x
-    cat <<'EOF'
-
-You must enable the Ubuntu Universe repository before installing
-Zulip.  You can do this with:
-
-    sudo add-apt-repository universe
-    sudo apt update
-
-For more information, see:
-  https://zulip.readthedocs.io/en/latest/production/requirements.html
-EOF
-    exit 1
-fi
+source "$ZULIP_PATH/scripts/lib/install-util"
```

Comment:
This line fails the shellcheck with an error of
`Can't follow non-constant source. Use a directive to specify location. (w1090)`. @andersk do you have a solution for this? `install-util` contains several useful functions that are used by `./scripts/lib/install`.

Reply:
Am using `# shellcheck source=/dev/null` for now

Reply:
I do, actually: #13546. However, as per Tim’s earlier [comment](https://github.com/zulip/zulip/pull/11302#issuecomment-522264690), indirecting to a separate file for a handful of straight-line functions each invoked once from one script doesn’t actually seem useful here. I think it would be clearer to do the operations inline in `scripts/lib/install`.

Reply:
OK, I will remove the function indirections.